### DISCUSSION TITLE
Change Network Name

### DISCUSSION THREAD

**Discussion by hpacherkar** [srsran/srsRAN_Project#346] [General Help] [answered]
Hello, I am currently using srsran gnb with our own 5G network. The network name in the COTs UE shows up as Open5Gs. Is there way i can change that ?

**Comment by AzeezEbrahim** [comment]
yes you can, look for `amf.yaml` file

**Reply by hpacherkar** [reply]
There is no amf.yaml in srsRAN_Project. Also I am not using Open5GS. Still the network name shows up as Open5gs in the phone.

**Reply by robertfalkenberg** [reply, marked as answer]
Maybe the COTS phone just recalls the network name from an earlier attempt (same PLMN) with open5GS?
The name may become updated after successful connection to the other network if it provides a different name.

In any case, this is independent from the gNB.
